### PR TITLE
Adding Grift

### PR DESCRIPTION
**Pull request by deyaaeldeen**

### CODE REVIEW

**Comment by deyaaeldeen** [comment]
@akuhlens, would you review it first?

File: templates/publications.rkt [hunk 1]
```diff
@@ -35,3 +49,10 @@
     #:author (list ben-greenman zeina-migeed)
     #:href "https://popl18.sigplan.org/event/pepm-2018-on-the-cost-of-type-tag-soundness"
     #:venue pepm-2018))
+
+(define kas-scheme-2018
```
Comment (bennn) [inline templates/publications.rkt:53]: these should go on the main page too (`index.rkt`)

Comment (deyaaeldeen) [reply templates/publications.rkt:53]: done.

File: index.rkt [hunk 2]
```diff
@@ -66,6 +66,7 @@
   }}}
   @div[class: "row"]{
     @research-highlights[
+      @to-appear[@kas-scheme-2018]
```
Comment (bennn) [inline index.rkt]: Please move this down to line 80 --- after the SPLASH papers --- so everything's in chronological order.

Also `@to-appear` can be `@appeared` :)

Comment (deyaaeldeen) [reply index.rkt]: done.

**Review by bennn — APPROVED** [review]
I can merge this if you're ready, otherwise I'll wait until Andre takes a look

**Review by akuhlens — CHANGES_REQUESTED** [review]
Typo: in compare to => in comparison to

I prefer that we use the wording "performance in the ballpark of". We are not trying to compete with ocaml, racket, chez scheme, or gambit.

Otherwise this looks good.

**Comment by deyaaeldeen** [comment]
Done. @bennn, now it is ready to be merged.

**Pull request merged**